sound-bubble/imports/ui/stylesheets/main.css

```diff
@@ -519,8 +506,10 @@ padding:0;
   border:1px solid #34363b;
   float:right;
   background:url(https://media1.giphy.com/media/l3q2Xl2Rd8wZfMCaI/200w.webp?cid=790b76115d3e6c3732544e6577a43431&rid=200w.webp) repeat;
+  overflow-wrap:break-word;
 }
 
```

Comment:
Comments shows when hover the user icon components, but user icon only gets bigger when hover inside the padding. There should be a 0 padding or 0 margin to keep the icon enlarge when hover and show the comments

Reply:
This seems to be caused by "margin-right: 1em;" on user-comment-image, but removing it changes the comments alignment a bit.

Reply:
fixed